### PR TITLE
docs: add carbon svelte implementation

### PR DESCRIPTION
**Pull request by metonym**
Closes #

This PR introduces Svelte as a language/framework that implements Carbon. It is akin to Angular and Vue in that it is community supported.

#### Changelog

**New**

- add Svelte as a new implementation

**Changed**

- {{change thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

**Review by dakahn — APPROVED** [review]
seems good to me -- did we want to also add it under the description in Other Frameworks tab?

**Comment by metonym** [comment]
@dakahn Good catch – added.

File: src/pages/developing/frameworks/other-frameworks.mdx [hunk 1]
```diff
@@ -2,15 +2,16 @@
...
+  the system in vanilla JS, Angular, Vue, and Svelte.
+tabs: ['React', 'Vanilla', 'Angular', 'Vue', 'Svelte', 'Other frameworks']
 ---
 
 <PageDescription>
 
 The Carbon Design System is built React first. We also support core parts of the
-system in vanilla JS, Angular, and Vue. If you’re you’re using a different
-framework, you can still build components by following the guidelines below.
+system in vanilla JS, Angular, Vue and Svelte. If you’re you’re using a
```
Comment (janchild) [inline src/pages/developing/frameworks/other-frameworks.mdx:12]:
```suggestion
system in vanilla JS, Angular, Vue, and Svelte. If you’re using a
```

File: src/pages/developing/frameworks/react.mdx [hunk 2]
```diff
@@ -2,16 +2,16 @@
...
+tabs: ['React', 'Vanilla', 'Angular', 'Vue', 'Svelte', 'Other frameworks']
 ---
 
 <PageDescription>
 
-The Carbon React library provides front-end developers & engineers a collection of reusable
-React components to build websites and user interfaces. Adopting the library
-enables developers to use consistent markup, styles, and behavior in prototype
-and production work.
+The Carbon React library provides front-end developers & engineers a collection
```
Comment (janchild) [inline src/pages/developing/frameworks/react.mdx:11]:
```suggestion
The Carbon React library provides front-end developers and engineers a collection
```

**Review by janchild — COMMENTED** [review]
Thanks! Looks good. Just a couple of copyedits needed here.

**Review by tw15egan — APPROVED** [review]
Looks great! 🎉 👍 ✅

**Review by janchild — APPROVED** [review]
Looking great! :tada:

**Pull request merged**